src/features/slackapi/slackapi-utils.ts

```diff
@@ -98,22 +106,25 @@ Have a great rest of the day!
 
     const customMessage = `
 Hi ${firstName},
-Last week (week: ${ week }, ${startDate} - ${endDate}) you worked ${logged} with an expected time of ${expected}.
+Last week (week: ${week}, ${startDate} - ${endDate}) you worked ${logged} with an expected time of ${expected}.
 ${message}
-Project time: ${project}, Internal time: ${internal}.
+Logged project time: ${loggedProject}, Billable project time: ${billableProject}, Non billable project time: ${nonBillableProject}, Internal time: ${internal}.
 Your percentage of billable hours was: ${billableHoursPercentage}%
 You ${+billableHoursPercentage >= 75 ? "worked the target 75% billable hours last week:+1:" : "did not work the target 75% billable hours last week:-1:"}.
```

Comment:
Depending on how the minimum billable rate is handled (see previous comment for line 60). This could be left as is, or calculated from the minimumBillable rate from timebank